### ISSUE TITLE
[CoE Starter Kit - BUG] PowerBI CoE Dashboard Inventory - Some of the AI Builder Model Values does not show the Maker Details

### ISSUE THREAD

**Issue by JanusAllenCapili** [microsoft/coe-starter-kit#8633]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Hi PowerCAT Team,

I would like to check if this is a bug or it might be a setting on some of the CoE components that needs to be checked to remediate this concern.

I noticed on our Inventory report that some of the AI Builder Model fields has blank values for the Maker. Sample screenshot as illustrated below:

![image](https://github.com/microsoft/coe-starter-kit/assets/156774820/c993f9b5-877c-4c25-ad77-b48e8fd60ac4)

Other AI Builder Models show the maker who created the AI Model. So it is expected that it should also show for these AI Builder Models.

Appreciate your inputs.

### Expected Behavior

It is expected that it will show the Maker who created the AI Builder Model.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.32.2

### What app or flow are you having the issue with?

N/A

### What method are you using to get inventory and telemetry?

None

AB#3814

**Comment by Jenefer-Monroe** [comment]
Is this a custom report? 
If I look in the Power Platform Admin View app, I do see blanks for some models in my test tenant but they are for orphaned cases. So cases where we don't know the owner for some reason, typically because they left the organization before the content was inventoried.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/4b25452d-50c3-4e5f-b38e-fb29fe13bf5a)

**Comment by JanusAllenCapili** [comment]
Hi @Jenefer-Monroe ,

Thanks for your response.

The report is coming from the Prebuilt PowerBI CoE Production Inventory. I added additional columns on the AI Builder Models report to identify the Maker who created the AI Builder Model as well.

I checked on the Power Platform Admin View > AI Models, and I see some entries in "blank" under the "Ai Creator", and valued as "Yes" under the Ai is Orphaned column:

![image](https://github.com/user-attachments/assets/c1bacbda-1e1a-4b8f-bd91-09cdc92f58be)

It seems that the "Ai is Orphaned" column is not available yet on the "AI Builder Model" table. It might also be a good feature to add the column, "AI is Orphaned" into the next version of the PowerBI CoE Reports, as this would really be helpful to identify if an AI Builder Model is orphaned for cases such as clean-up processes.
